src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -71,9 +72,10 @@ public void parseCommand_exit() throws Exception {
 
     @Test
     public void parseCommand_find() throws Exception {
-        List<String> keywords = Arrays.asList("foo", "bar", "baz");
+        List<String> keywords = Arrays.asList("Assignment", "homework", "lab");
         FindCommand command = (FindCommand) parser.parseCommand(
-                FindCommand.COMMAND_WORD + " " + keywords.stream().collect(Collectors.joining(" ")));
+                FindCommand.COMMAND_WORD + " " + PREFIX_NAME + keywords.stream()
+                        .collect(Collectors.joining(" ")));
         assertEquals(new FindCommand(new NameContainsKeywordsPredicate(keywords)), command);
     }
```

Comment:
You may want to consider testing for the other fields as well to ensure that they are working as they are supposed to but this is a minor nitpick :)

Reply:
Amended!